src/main/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSink.java

```diff
@@ -54,19 +58,31 @@ public static void setHeader(final SAMFileHeader header) {
      */
     public static void writeReads(
             final JavaSparkContext ctx, final String outputFile, final JavaRDD<GATKRead> rddReads,
-            final SAMFileHeader header, boolean singleFile) throws IOException {
+            final SAMFileHeader header, boolean singleFile, boolean parquet) throws IOException {
```

Comment:
Where is the idiomatic place to put such an `enum`?  How about `ReadsWriteFormat` for a name?